client/__tests__/e2e/puppeteer.setup.ts

```diff
@@ -1,107 +0,0 @@
-      interceptedRequest.respond({
-        status: 200,
-        contentType: "application/json",
-        body: JSON.stringify(DATASET_METADATA_RESPONSE),
-        // (thuang): Add headers so FE@localhost:3000 can access API@localhost:5000
-        headers: {
-          "Access-Control-Allow-Origin": referer.slice(0, referer.length - 1),
-          "Access-Control-Allow-Credentials": "true",
-        },
-      });
-      return;
-    } if (interceptedRequest.url().endsWith("/config")) {
-      const { referer } = interceptedRequest.headers();
-      
-      fetch(interceptedRequest.url()).then((response: any) => {
-        response.json().then((body: any) => {
-          interceptedRequest.respond({
-            status: 200,
-            contentType: "application/json",
-            body: JSON.stringify({
-              config: {
-                ...body.config,
-                links: {
-                  ...body.config.links,                
-                  "collections-home-page": "https://cellxgene.cziscience.com/dummy-collection",
-                },              
-              }
-            }),
-            headers: {
-              "Access-Control-Allow-Origin": referer.slice(0, referer.length - 1),
-              "Access-Control-Allow-Credentials": "true",
-            },
-          });
-        });
-      });
-      return;      
-    }
-    interceptedRequest.continue();
-  });
-
-  const userAgent = await browser.userAgent();
-  await page.setUserAgent(`${userAgent}bot`);
-
-  // @ts-expect-error ts-migrate(2341) FIXME: Property '_client' is private and only accessible ... Remove this comment to see the full error message
-  await page._client.send("Animation.setPlaybackRate", { playbackRate: 12 });
-
-  page.on("pageerror", (err) => {
-    throw new Error(`Console error: ${err}`);
-  });
-
-  page.on("error", (err) => {
-    throw new Error(`Console error: ${err}`);
-  });
-
-  page.on("console", async (msg) => {
-    if (isDev || isDebug) {
-      // If there is a console.error but an error is not thrown, this will ensure the test fails
-      console.log(`PAGE LOG: ${msg.text()}`);
-      if (msg.type() === "error") {
-        // TODO: chromium does not currently support the CSP directive on the
-        // line below, so we swallow this error. Remove this when the test
-        // suite uses a browser version that supports this directive.
-        if (
-          msg.text() ===
-          "Unrecognized Content-Security-Policy directive 'require-trusted-types-for'.\n"
-        ) {
-          return;
-        }
-        const errorMsgText = await Promise.all(
-          // TODO can we do this without internal properties?
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-          msg.args().map((arg: any) => arg._remoteObject.description)
-        );
-        throw new Error(`Console error: ${errorMsgText}`);
-      }
-    }
-  });
```

Comment:
old error handling

Reply:
Sorry I forgot - did you create a ticket to capture the work to reintroduce this 😄 ? Thanks!

Reply:
https://github.com/chanzuckerberg/single-cell-explorer/issues/719

Reply:
Thanks! Linked to the visium epic 👍